docs/README.md

```diff
@@ -20,11 +20,14 @@ feature:
 
 user:
   title: Who is using SkyWalking
-  content: Various companies and organizations use SkyWalking for research, production and commercial products. 
+  content: Various companies and organizations use SkyWalking for research, production and commercial products.
 
 news:
   title: Events & News
   content:
+  - header: Welcome Zhusheng Xu as new committer
+    content: Based on his continuous contributions, Zhusheng Xu (a.k.a [aderm](https://github.com/aderm)) has been voted as a new committer.
+    time: Feb. 21st, 2020
```

Comment:
We only keep 4 news, please remove the oldest one.

Reply:
ok